<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addproduct</class>
 <widget class="QDialog" name="Addproduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_pid">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Product ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 10pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>523</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_productid">
      <property name="styleSheet">
       <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="styleSheet">
       <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_price">
      <property name="styleSheet">
       <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_quantity">
      <property name="styleSheet">
       <string notr="true">background-color:white;
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border: 1px solid black;
border-radius: 3px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>135</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_addproduct">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(18,10,131);
font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
border: 1px;
border-radius: 3px;
</string>
      </property>
      <property name="text">
       <string>ADD PRODUCTS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 0, 0);
color: white;
font: 700 10pt &quot;Segoe UI&quot;;
border: 1pz solid black;
</string>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
